tools/cythonize.py

```diff
@@ -100,6 +100,24 @@ def process_tempita_pyx(fromfile, tofile):
         f.write(pyxcontent)
     process_pyx(pyxfile, tofile)
 
+
+def process_tempita_pxi(fromfile, tofile):
+    try:
+        try:
+            from Cython import Tempita as tempita
```

Comment:
I'm a little confused by this statement here.  `Tempita` has always been assumed to be imported from Cython.